src/Entity/Search.php

```diff
@@ -55,14 +56,15 @@ class Search
     private $category;
 
     /**
-     * @ORM\ManyToMany(targetEntity="App\Entity\Material", mappedBy="searches")
+     * @ORM\ManyToMany(targetEntity="App\Entity\Material", mappedBy="searches", fetch="EXTRA_LAZY")
      *
      * @Groups({"material"})
      */
     private $materials;
 
     /**
-     * @ORM\OneToMany(targetEntity="App\Entity\SearchRun", mappedBy="search", orphanRemoval=true)
+     * @ORM\OneToMany(targetEntity="App\Entity\SearchRun", mappedBy="search", orphanRemoval=true, fetch="EXTRA_LAZY")
+     * @ORM\OrderBy({"id" = "DESC"})
      */
     private $searchRuns;
```

Comment:
What is a searchRuns?

Reply:
Each time the Search is performed and materials saved, a "SearchRun" is registered with datetime, success, error if fail